CONTRIBUTING

```diff
@@ -0,0 +1,118 @@
+# Contributing to the CNTI
+
+Welcome to the Cloud Native Telecom Initiative (CNTi). This is an open, public project welcoming anyone who would like to help in one or more of our focus areas.  Our goal is to advance cloud native networking including developing best practices, maintaining a testing catalog, and providing vendor neutral certification.  We're glad you're here!
+
+To learn more about our focus areas and our different collaboration tools, please visit our focus area wiki pages:
+
+- [Best Practices](https://wiki.lfnetworking.org/x/HAAxBw)
+
+- [Test Catalog](https://wiki.lfnetworking.org/x/HgAxBw)
+
+- [Certification](https://wiki.lfnetworking.org/x/IAAxBw)
+
```

Comment:
Do we want to include crystal libs?
https://github.com/cnf-testsuite/
Often we have direct test functionality dependencies there.

Reply:
Good point, however, the links are pointing to the wiki page for respective focus areas.  On those wiki pages we have a link to our repositories.  I would suggest we update this page to include the other repos where we have dependencies for test suite.  See here -> https://wiki.lfnetworking.org/display/LN/Repositories+and+Project+Backlogs